velox/exec/Spill.h

```diff
@@ -125,6 +127,9 @@ class SpillMergeStream : public MergeStream {
 
   // Covers all rows inn 'rowVector_' Set if 'decoded_' is non-empty.
   SelectivityVector rows_;
+
+  // True if the stream is closed.
+  bool closed_;
```

Comment:
When closed_ is set?
```
bool closed_{false}
```
Move to first? And shall we add to check in API that closed_ is not set?